CodeStarterCamp_Week4/body.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  body.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by minsong kim on 2023/03/13.
+//
+
+import Foundation
+
+class BodyCondition {
+    var upperBodyStrength: Int
+    var lowerBodyStrength: Int
+    var muscularEndurance: Int
+    var fatigability: Int
+    
+    init(upperBodyStrength: Int, lowerBodyStrength: Int, muscularEndurance: Int, fatigability: Int) {
+        self.upperBodyStrength = upperBodyStrength
+        self.lowerBodyStrength = lowerBodyStrength
+        self.muscularEndurance = muscularEndurance
+        self.fatigability = fatigability
+    }
+    
+    func conditionCheck() {
+        print(
+        """
+        --------------
+        현재의 컨디션은 다음과 같습니다.
+        상체근력: \(upperBodyStrength)
+        하체근력: \(lowerBodyStrength)
+        근지구력: \(muscularEndurance)
+        피로도: \(fatigability)
+        --------------
+        """
+        )
+    }
+}
+
+struct Activity {
+    let name: String
+    let action: (BodyCondition) -> Void
+}
+
+
+let 윗몸일으키기: Activity = Activity(name: "윗몸일으키기", action: { BodyCondition -> Void in
+    let upperBodyCount = Int.random(in: 10...20)
+    let fatigabilityCount = Int.random(in: 10...20)
+    BodyCondition.upperBodyStrength += upperBodyCount
```

Comment:
요건 사람마다 다르겠습니다만, 저는 선언부와 로직을 분리하는게 읽기 더 좋더라구요! 😁
참고하시면 좋을거 같습니다!
```suggestion
    let fatigabilityCount = Int.random(in: 10...20)
    
    BodyCondition.upperBodyStrength += upperBodyCount
```